extensions/brew/package.json

```diff
@@ -56,6 +57,23 @@
       "required": false,
       "title": "Custom Brew Executable Path",
       "description": "Set this if your brew executable is in a non-standard location"
+    },
+    {
+      "name": "terminalApp",
+      "type": "dropdown",
+      "required": false,
+      "title": "Choose Terminal Application",
```

Comment:
Maybe just "Terminal Application" for the title?

Reply:
```suggestion
      "title": "Terminal Application",
```
👍